composer.json

```
@@ -3,47 +3,73 @@
   "type": "project",
   "license": "GPL-2.0-or-later",
+  "homepage": "https://www.drupal.org/project/apigee_devportal_kickstart",
+  "support": {
+    "docs": "https://www.drupal.org/docs/8/modules/apigee-developer-portal-kickstart"
+  },
   "repositories": [
     {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
     }
   ],
-  "config": {
-    "process-timeout": 0
-  },
   "require": {
     "composer/installers": "^1.2",
-    "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
+    "drupal/core-composer-scaffold": "^8.8",
+    "drupal/core-project-message": "^8.8",
+    "drupal/core-recommended": "^8.8",
     "apigee/apigee_devportal_kickstart": "^1.0.0",
     "drupal/apigee_m10n": "^1.0.0",
     "drupal/commerce": "^2.13",
-    "php": "^7.1",
+    "php": "^7.2"
+  },
+  "require-dev": {
+    "drupal/core-dev": "^8.8",
     "drush/drush": "^9.7"
   },
+  "conflict": {
+    "drupal/drupal": "*"
+  },
   "minimum-stability": "dev",
   "prefer-stable": true,
-  "scripts": {
-    "post-install-cmd": [
-      "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
-    ],
-    "post-update-cmd": [
-      "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
-    ],
-    "quick-start": [
-      "composer install",
-      "php web/core/scripts/drupal quick-start apigee_devportal_kickstart"
-    ]
+  "config": {
+    "sort-packages": true
   },
   "extra": {
+    "drupal-scaffold": {
+      "locations": {
+        "web-root": "web/"
+      }
+    },
     "installer-paths": {
       "web/core": ["type:drupal-core"],
       "web/libraries/{$name}": ["type:drupal-library"],
       "web/modules/contrib/{$name}": ["type:drupal-module"],
       "web/profiles/contrib/{$name}": ["type:drupal-profile"],
-      "web/themes/contrib/{$name}": ["type:drupal-theme"]
+      "web/themes/contrib/{$name}": ["type:drupal-theme"],
+      "drush/Commands/contrib/{$name}": ["type:drupal-drush"],
+      "web/modules/custom/{$name}": ["type:drupal-custom-module"],
+      "web/themes/custom/{$name}": ["type:drupal-custom-theme"]
     },
-    "enable-patching": true
+    "drupal-core-project-message": {
+      "include-keys": ["homepage", "support"],
+      "post-create-project-cmd-message": [
+        "<bg=blue;fg=white>                                                                                 </>",
+        "<bg=blue;fg=white>  Congratulations, you’ve installed the Apigee Developer Portal Drupal codebase  </>",
+        "<bg=blue;fg=white>  from the apigee/devportal-kickstart-project template!                          </>",
+        "<bg=blue;fg=white>                                                                                 </>",
+        "",
+        "<bg=yellow;fg=black>Next steps</>:",
+        "  * Install the site: https://www.drupal.org/docs/8/modules/apigee-developer-portal-kickstart/use-kickstart-with-apigee-edge-for-private-cloud",
```

I put info on all ways to install on this page to be more generic: 
```suggestion
        "  * Install the site: https://www.drupal.org/docs/8/modules/apigee-developer-portal-kickstart/get-started-with-kickstart",
```